torch_xla/csrc/tensor_methods.cpp

```diff
@@ -2567,10 +2567,10 @@ void uniform_(XLATensorPtr& input, double from, double to) {
 }
 
 XLATensorPtr unsqueeze(const XLATensorPtr& input, int64_t dim) {
-  auto input_shape = input->shape();
+  xla::util::MaybeRef<xla::Shape> input_shape = input->shape();
```

Comment:
Why this would fix anything?

Reply:
It doesn't fix anything. I find it easier to read the code when I make the type explicit.

Reply:
hahaha, then you must hate python 😜 It's up to you. I'm fine with it.